shell/browser/api/atom_api_web_contents.cc

```diff
@@ -458,6 +468,23 @@ void WebContents::InitWithSessionAndOptions(
   prefs->subpixel_rendering = params->subpixel_rendering;
 #endif
 
+#if defined(OS_MACOSX)
+  base::TimeDelta interval;
+  if (ui::TextInsertionCaretBlinkPeriod(&interval))
+    prefs->caret_blink_interval = interval;
+#elif defined(OS_LINUX)
+  views::LinuxUI* linux_ui = views::LinuxUI::instance();
+  if (linux_ui)
+    prefs->caret_blink_interval = linux_ui->GetCursorBlinkInterval();
+#endif
+
+#if defined(OS_WIN)
+  static const size_t system_value = ::GetCaretBlinkTime();
+  if (system_value != 0 && system_value != INFINITE)
+    prefs->caret_blink_interval =
+        base::TimeDelta::FromMilliseconds(system_value);
```

Comment:
_(minor)_ One thing that might be nice to copy from [TextField::GetCaretBlinkInterval](https://cs.chromium.org/chromium/src/ui/views/controls/textfield/textfield.cc?sq=package:chromium&dr=CSs&g=0&l=266) is that if `::GetCaretBlinkTime` fails, it uses a 500msec blink as a fallback

e.g.

```cpp
const auto system_msec = ::GetCaretBlinkTime();
if (system_msec == INFINITE)
  prefs->caret_blink_interval = base::TimeDelta()
else if (system_msec == 0)
  prefs->caret_blink_interval = base::TimeDelta::FromMilliseconds(500);
else
  prefs->caret_blink_interval = base::TimeDelta::FromMilliseconds(system_msec);
```